bowling.test.js

```diff
@@ -0,0 +1,54 @@
+const Bowling = require('./bowling')
+
+describe ('Bowling', () => {
+  it ('can make a new game', () => {
+    const game = new Bowling();
+  });
+
+  it ('can roll a gutter game', () => {
+    const game = new Bowling();
+    for (let i = 0; i < 20; i++) {
+      game.roll(0) 
+    }
+    expect(game.score()).toBe(0);
+  })
+
+  it ('can roll all 1s', () => {
+    const game = new Bowling();
+    for (let i = 0; i < 20; i++) {
+      game.roll(1);
+    }
+    expect(game.score()).toBe(20);
+  })
+
+  it ('can roll a spare', () => {
+    const game = new Bowling();
+    game.roll(5);
```

Comment:
It's a little hard for me to interpret tests like this, as I'm not sure where some of these numbers come from, eg i ranging up to 18, game.roll(5)